### PR TITLE
dev-python/lmdb: version bump to 1.0.0, update metadata github id

### PR DESCRIPTION
**Pull request by wimmuskee**
also add python_test and py3_8 python compat

Closes: https://bugs.gentoo.org/744334
Signed-off-by: Wim Muskee <wimmuskee@gmail.com>

### CODE REVIEW

File: dev-python/lmdb/lmdb-1.0.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+RDEPEND="dev-db/lmdb:="
+DEPEND="${RDEPEND}"
+BDEPEND="test? ( dev-python/pytest[${PYTHON_USEDEP}] )"
+
+python_compile() {
+	LMDB_FORCE_SYSTEM=1 distutils-r1_python_compile
+}
+
+python_test() {
```
Comment (DarthGandalf) [inline dev-python/lmdb/lmdb-1.0.0.ebuild:28]: Replace this and BDEPEND with `distutils_enable_tests pytest`

Comment (DarthGandalf) [reply dev-python/lmdb/lmdb-1.0.0.ebuild:28]: And it adds the IUSE and RESTRICT too

Comment (sbraz) [reply dev-python/lmdb/lmdb-1.0.0.ebuild:28]: Tests aren't named `test_*` so pytest with its default options doesn't detect them. I will be adding `distutils_enable_tests pytest` on merge to remove the need for RESTRICT, IUSE and BDEPEND.

**Comment by sbraz** [comment]
Thanks, pushed with a few changes.

**Pull request closed without merging**